scalableminds/webknossos
# Fix local to global layer index look up

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- check that datasets with heterogenous mags render correctly
- for the dataset in question (see slack thread), I checked that both layer orderings render okay

### Issues:
- fixes rendering bug introduced by #6748 
- also see https://scm.slack.com/archives/C5AKLAV0B/p1683617650361979

------
(Please delete unneeded items, merge only when none are left open)
